pkg/devspace/configure/deployment.go

```diff
@@ -76,6 +77,20 @@ func GetDockerfileComponentDeployment(config *latest.Config, generatedConfig *ge
 			return nil, nil, errors.Wrap(err, "parsing port")
 		}
 
+		if port < 1024 {
+			log.Warn("Your application listens on a system port [0-1024]. Choose a forwarding-port to access your application via localhost.")
+
+			portString := survey.Question(&survey.QuestionOptions{
+				Question:     "Which forwarding port [1024-49151] do you want to use to access your application?",
+				DefaultValue: strconv.Itoa(port + 8000),
+			})
+
+			port, err = strconv.Atoi(portString)
```

Comment:
This will potentially lead to wrong configuration since an application port often cannot chosen at will. Rather we should set the specified port here as the port of the service and the other port as the target port the service will map incoming connections onto

Reply:
That is true. This should be moved to init.addDevConfig and do what it says in the question. Done.